src/styles/main.scss

```diff
@@ -1,3 +1,65 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  gap: 1px;
+  width: 726px;
```

Comment:
It's better to create sass function and calculate this width instead of hardcode it